cadc-log/src/main/java/ca/nrc/cadc/log/WebServiceLogInfo.java

```diff
@@ -165,7 +166,31 @@ private String getPreamble() {
         return sb.toString();
     }
 
-    private void populate(StringBuilder sb, Class c) {
+    // handle Lists, refer to GroupLogInfo.addedMembers
+    // currently only List<String> is handled
+    private void populateList(StringBuilder sb, List<?> o) {
+        sb.append("[");
+        if (!(o.isEmpty())) {
+            boolean isFirstElement = true;
+            if (o.get(0) instanceof String) {
```

Comment:
If this is not an instance of List<String> then the value is lost.  It should have the same behaviour as before if not an instance of List<String>.

Reply:
o has to be an instance of List at this point.

Reply:
I mean the case where o is, for example, an instance of List<Integer>.  Why not just check for List<String> instead of List<?> in populate()?